<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddClientDialog</class>
 <widget class="QDialog" name="AddClientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Замовлення</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>691</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Номер телефону</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="phoneNumberEdit"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="emailEdit"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="firstNameEdit"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="surnameEdit"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="middleNameEdit"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="rentedCarEdit"/>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="pbAcceptClient">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Електронна пошта</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Ім'я</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Прізвище</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>По-батькові</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Вибраний автомобіль:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
